.infrastructure/55_cognito.tf

```
@@ -0,0 +1,85 @@
+resource "aws_cognito_user_pool" "devportal" {
+  name = "devportalpool"
```

Add [deletion_protection](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cognito_user_pool#deletion_protection) parameter

```suggestion
  name                = "devportalpool"
  deletion_protection = "ACTIVE"

```